client/components/LearnerViewDirective.js

```diff
@@ -35,8 +35,10 @@ tie.directive('learnerView', [function() {
                   </div>
                   <div ng-class="{'tie-step-line': $index < (questionIds.length - 1)}"></div>
                 </div>
+                <div ng-class="{'tie-step-line': $index < (questionIds.length - 1)}"></div>
               </div>
             </div>
+          </div>
```

Comment:
This div tag on line 40 matches with the one on line 26(div class="tie-question-ui-inner")